sql_prep/models/achievements/agg__napul_cesty.sql

```diff
@@ -0,0 +1,39 @@
+WITH ib_base AS (
+    SELECT * FROM {{ ref('base__debater_ib') }}
+),
+
+aggregate_ib AS (
+    SELECT
+        clovek_id,
+        ANY_VALUE(school_year) as school_year,
```

Comment:
Why the ANY_VALUE? Wouldn't it make sense to skip it altogether if you don't want to aggregate by it?